components/blitz/src/omero/cmd/mail/SendEmailRequestI.java

```diff
@@ -74,17 +74,13 @@ public SendEmailRequestI(MailUtil mailUtil) {
     public void init(Helper helper) {
         this.helper = helper;
 
-        final EventContext ec = ((LocalAdmin) helper.getServiceFactory()
```

Comment:
as discussed with @joshmoore I will add MailUtil to ShareBean to send email in a separate thread. In 5.1 we will not be getting notifications for emails when sharing as it is low priority task.